### ISSUE TITLE
[REQ] Forgot password

### ISSUE THREAD

**Issue by nicolemorse** [aonez/Keka#1500]
### Is your feature request related to a problem? Please describe
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...].
I forgot my password for Keka
### Describe the solution you'd like
How can I change a forgotten password in Keka?

**Comment by aonez** [comment]
Hi! Are you talking about the default password set in Keka?

**Comment by nicolemorse** [comment]
Yes. I can’t open my zip files because I forgot my password. Is there a way to reset it?Blessings, Nicole Morse On 16 Aug 2024, at 08:48, aONe ***@***.***> wrote:﻿
Hi! Are you talking about the default password set in Keka?

**Comment by nicolemorse** [comment]
Is there a solution? Thanks for your help. Blessings, Nicole Morse On 16 Aug 2024, at 10:46, Nicole Morse ***@***.***> wrote:﻿Yes. I can’t open my zip files because I forgot my password. Is there a way to reset it?Blessings, Nicole Morse On 16 Aug 2024, at 08:48, aONe ***@***.***> wrote:﻿
Hi! Are you talking about the default password set in Keka?

**Comment by aonez** [comment]
You can check your password in the Keychain:

<img width="646" alt="Screenshot 2024-08-21 at 09 36 58" src="https://github.com/user-attachments/assets/3dae5d43-4025-4c93-ab51-e6fc5ab6e8a3">
